src/app/calc.css

```diff
@@ -0,0 +1,26 @@
+#head{
```

Comment:
use prettier on css files as well. Should be:

```suggestion
#head {
```

Reply:
Check ✅